projects/1.18/assets/expanded-caves/expcaves/lang/zh_cn.json

```
@@ -0,0 +1,191 @@
+  "block.expcaves.dirtstone_button": "泥土岩按钮",
+  "block.expcaves.dirtstone_cobble": "泥土岩圆石",
+  "block.expcaves.dirtstone_cobble_button": "泥土岩圆石按钮",
+  "block.expcaves.dirtstone_cobble_pressure_plate": "泥土岩圆石压力板",
+  "block.expcaves.dirtstone_cobble_slab": "泥土岩圆石台阶",
+  "block.expcaves.dirtstone_cobble_stairs": "泥土岩圆石楼梯",
+  "block.expcaves.dirtstone_cobble_wall": "泥土岩圆石墙",
+  "block.expcaves.dirtstone_pressure_plate": "泥土岩压力板",
+  "block.expcaves.dirtstone_slab": "泥土岩台阶",
+  "block.expcaves.dirtstone_stairs": "泥土岩楼梯",
+  "block.expcaves.dirtstone_wall": "泥土岩墙",
+  "block.expcaves.dry_moss": "干枯苔藓",
+  "block.expcaves.fire_moss": "火焰苔藓",
+  "block.expcaves.fluoshroom": "荧光蘑菇",
+  "block.expcaves.frozen_moss": "寒冰苔藓",
+  "block.expcaves.goldishroom": "金蘑菇",
+  "block.expcaves.granite_pebble": "花岗岩石子",
+  "block.expcaves.granite_stalactite": "花岗岩钟乳石",
+  "block.expcaves.granite_stalagmite": "花岗岩石笋",
+  "block.expcaves.granite_tall_stalactite": "高花岗岩钟乳石",
+  "block.expcaves.granite_tall_stalagmite": "高花岗岩石笋",
+  "block.expcaves.hanging_roots": "倒悬根茎",
+  "block.expcaves.humid_moss_ceiling": "湿润苔藓",
+  "block.expcaves.humid_moss_ground": "湿润苔藓",
+  "block.expcaves.lavastone": "熔岩石",
+  "block.expcaves.lavastone_button": "熔岩石按钮",
+  "block.expcaves.lavastone_pebble": "熔岩石石子",
+  "block.expcaves.lavastone_pressure_plate": "熔岩石压力板",
+  "block.expcaves.lavastone_slab": "熔岩石台阶",
+  "block.expcaves.lavastone_stairs": "熔岩石楼梯",
+  "block.expcaves.lavastone_stalactite": "熔岩石钟乳石",
+  "block.expcaves.lavastone_stalagmite": "熔岩石笋",
+  "block.expcaves.lavastone_tall_stalactite": "高熔岩石钟乳石",
+  "block.expcaves.lavastone_tall_stalagmite": "高熔岩石笋",
+  "block.expcaves.lavastone_wall": "熔岩石墙",
+  "block.expcaves.lumishroom": "夜明蘑菇",
+  "block.expcaves.marlstone": "泥灰岩",
+  "block.expcaves.marlstone_button": "泥灰岩按钮",
+  "block.expcaves.marlstone_powder": "泥灰岩粉末",
+  "block.expcaves.marlstone_pressure_plate": "泥灰岩压力板",
+  "block.expcaves.marlstone_wall": "泥灰岩墙",
+  "block.expcaves.netherrack_pebble": "地狱岩石子",
+  "block.expcaves.netherrack_stalactite": "地狱岩钟乳石",
+  "block.expcaves.netherrack_stalagmite": "地狱岩石笋",
+  "block.expcaves.netherrack_tall_stalactite": "高地狱岩钟乳石",
+  "block.expcaves.netherrack_tall_stalagmite": "高地狱岩石笋",
+  "block.expcaves.packed_ice_stalactite": "冰钟乳石",
+  "block.expcaves.packed_ice_stalagmite": "冰笋",
+  "block.expcaves.packed_ice_tall_stalactite": "高冰钟乳石",
+  "block.expcaves.packed_ice_tall_stalagmite": "高冰笋",
+  "block.expcaves.polished_lavastone": "磨制熔岩石",
+  "block.expcaves.polished_lavastone_button": "磨制熔岩石按钮",
+  "block.expcaves.polished_lavastone_pressure_plate": "磨制熔岩石压力板",
+  "block.expcaves.polished_lavastone_slab": "磨制熔岩石台阶",
+  "block.expcaves.polished_lavastone_stairs": "磨制熔岩石楼梯",
+  "block.expcaves.polished_lavastone_wall": "磨制熔岩石墙",
+  "block.expcaves.quartz_treasure_amphora": "石英宝藏罐",
+  "block.expcaves.rock_flint": "燧石",
+  "block.expcaves.rockshroom": "岩石蘑菇",
+  "block.expcaves.sediment_stone": "沉积岩",
```

```suggestion
  "block.expcaves.sediment_stone": "沉积岩",
```
正确的沉积岩得是 **sedimentary rock**
原文：**Sediment Stone**
| 严重程度 |
|--------------|
| ℹ️ 信息 / 不阻止合并  |

---

```suggestion
  "block.expcaves.sediment_stone": "沉积石",
```

---

沉积石搜不出东西来。
但是，原文错我们也得跟着错。